consensus/src/state_computer.rs

```diff
@@ -66,15 +69,18 @@ impl StateComputer for ExecutionProxy {
         &self,
         blocks: &[Arc<ExecutedBlock>],
         finality_proof: LedgerInfoWithSignatures,
+        callback: StateComputerCommitCallBackType,
     ) -> Result<(), ExecutionError> {
         let mut block_ids = Vec::new();
         let mut txns = Vec::new();
         let mut reconfig_events = Vec::new();
+        let mut blocks_clone = Vec::new();
```

Comment:
no need to clone, just change the loop to use &blocks?

Reply:
resolved